src/main/java/migrator/app/domain/index/IndexRepository.java

```diff
@@ -0,0 +1,74 @@
+package migrator.app.domain.index;
+
+import java.util.Collection;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Map;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import migrator.app.domain.table.model.Index;
+import migrator.app.migration.model.ChangeCommand;
+import migrator.app.migration.model.IndexProperty;
+import migrator.lib.repository.UniqueRepository;
+
+public class IndexRepository extends UniqueRepository<Index> {
```

Comment:
Add List<Repository<T>> indexes - list of indexes that will apply to add remove addAll methods